changelog.d/20250325_182401_beth_add_required_mfa_to_auth_policies.rst

```diff
@@ -0,0 +1,7 @@
+Changed
+~~~~~~~
+
+- Added the optional ``required_mfa`` field to ``AuthClient.create_policy()`` and
+  ``AuthClient.update_policy()`` request bodies. (:pr:`NUMBER`)
+- Updated the other _testing auth policy responses to include the additional
+  field, ``required_mfa``. (:pr:`NUMBER`)
```

Comment:
I think I prefer to omit this from the changelog, though it's arguable either way. `globus_sdk._testing` is public, but we usually don't put data changes there into the changelog.

```suggestion
```